stacks/upload-api-stack.js

```diff
@@ -146,6 +146,7 @@ export function UploadApiStack({ stack, app }) {
             INDEXING_SERVICE_URL,
             INDEXING_SERVICE_PROOF,
             HOSTED_ZONE: hostedZone ?? '',
+            PRINCIPAL_MAPPING: process.env.PRINCIPAL_MAPPING ?? ''
```

Comment:
could you add an example of what this should look like in the docs and the env.tpl?

https://github.com/storacha/w3infra/blob/main/README.md#environment-variables
https://github.com/storacha/w3infra/blob/main/.env.tpl